File: todo/app/src/main/java/com/k/todo/service/music/MPlayer.java
package com.k.todo.service.music;


import android.support.annotation.Nullable;

public interface MPlayer {

    void setDataSource(String path);

    void setNextDataSource(@Nullable String path);

    boolean isPlaying();

    boolean isInitialized();

    void play();

    void pause();

    void seekTo(int to);

    void setVolume(float vol);

    void setAudioSessionId(int sessionId);

    int getAudioSessionId();

    int duration();

    int position();

    void stop();

    void release();

}

File: todo/app/src/main/java/com/k/todo/service/music/MPlayer.kt
package com.k.todo.service.music

interface MPlayer {

    val isPlaying: Boolean

    val isInitialized: Boolean

    var audioSessionId: Int

    fun setDataSource(path: String)

    fun setNextDataSource(path: String?)

    fun play()

    fun pause()

    fun seekTo(to: Int)

    fun setVolume(vol: Float)

    fun duration(): Int

    fun position(): Int

    fun stop()

    fun release()

}
